tests/test_linear_models/test_linear_models.py

```diff
@@ -125,15 +125,13 @@ def reset_exported_models_directory(self):
     def test_full(self):
         self.reset_exported_models_directory()
         for category in self.LINEAR_MODELS.keys():
-            category_all_test_pass = True
             for model in self.LINEAR_MODELS[category]:
                 if isinstance(model, tuple):
                     func, model_name = model
                     if func == None:
                         print("Model: " + model_name + " is not supported in this python version.")
                         continue
-                category_all_test_pass = category_all_test_pass and model()
-                self.assertTrue(category_all_test_pass)
+                model()
```

Comment:
Can you explain why this change is required?

Reply:
Pytest-associated test functions are warned to return values and they will not be supported in the next versions of Pytest, so as you can see the `model` which is the test function of the associated model, doesn't return anything and it just has an `assertion`, so I removed `category_all_test_pass ` variable.